examples/js/ShaderSkin.js

```diff
@@ -338,26 +338,26 @@ THREE.ShaderSkin = {
 
 			{
 
-			"passID": { type: "i", value: 0 },
+				"passID": { type: "i", value: 0 },
 
-			"tDiffuse"	: { type: "t", value: null },
-			"tNormal"	: { type: "t", value: null },
+				"tDiffuse"	: { type: "t", value: null },
+				"tNormal"	: { type: "t", value: null },
 
-			"tBlur1"	: { type: "t", value: null },
-			"tBlur2"	: { type: "t", value: null },
-			"tBlur3"	: { type: "t", value: null },
-			"tBlur4"	: { type: "t", value: null },
+				"tBlur1"	: { type: "t", value: null },
```

Comment:
The problem with this file is that even with these tabs is doens't outline correctly.

https://github.com/mrdoob/three.js/pull/6863/files#diff-a1806af1528eb3e8af7952cfb105a597R46
vs
https://github.com/mrdoob/three.js/pull/6863/files#diff-a1806af1528eb3e8af7952cfb105a597R43

the colon doesn't matchup compared to each other .